Constitution.txt

```diff
@@ -1,490 +1,264 @@
+The judicial Power of these United States, shall be vested in one supreme Court, and in such inferior Courts as the Congress may from time to time ordain and establish. The Judges, both of the supreme and inferior Courts, shall hold their Offices during good Behavior, and shall, at stated Times, receive for their Services, a Compensation, which shall not be diminished during their Continuance in Office.
 
 Section 2
-1: The judicial Power shall extend to all Cases, in Law and Equity, arising under this Constitution, the Laws of the United States, and Treaties made, or which shall be made, under their Authority;—to all Cases affecting Ambassadors, other public Ministers and Consuls;—to all Cases of admiralty and maritime Jurisdiction;—to Controversies to which the United States shall be a Party;—to Controversies between two or more States;—between a State and Citizens of another State; —between Citizens of different States, —between Citizens of the same State claiming Lands under Grants of different States, and between a State, or the Citizens thereof, and foreign States, Citizens or Subjects.
+1: The judicial Power shall extend to all Cases, in Law and Equity, arising under this Constitution, the Laws of these United States, and Treaties made, or which shall be made, under their Authority;
+—to all Cases affecting Ambassadors, other public Ministers and Consuls;
+—to all Cases of admiralty and maritime Jurisdiction;
+—to Controversies to which these United States shall be a Party;
+—to Controversies between two or more States;
+—between a State and Citizens of another State;
+—between Citizens of different States,
+—between Citizens of the same State claiming Lands under Grants of different States, and between a State, or the Citizens thereof, and foreign States, Citizens or Subjects.
 
 2: In all Cases affecting Ambassadors, other public Ministers and Consuls, and those in which a State shall be Party, the supreme Court shall have original Jurisdiction. In all the other Cases before mentioned, the supreme Court shall have appellate Jurisdiction, both as to Law and Fact, with such Exceptions, and under such Regulations as the Congress shall make.
 
-3: The Trial of all Crimes, except in Cases of Impeachment, shall be by Jury; and such Trial shall be held in the State where the said Crimes shall have been committed; but when not committed within any State, the Trial shall be at such Place or Places as the Congress may by Law have directed.
+3: The supreme Court shall have the authority, with the assent of two-thirds of the Justices, to decline to hear a case over which they have appellate jurisdiction, or with unanimous assent, to decline to hear a case over which they have original jurisdiction; but if the Chief Executives, Attorneys General, or Legislative bodies of more than one-third of the States shall be Plaintiff to a case, the supreme Court shall hear the case.
```

Comment:
I formalize here constitutionally the process of Writs of Certiorari that has been established for the Supreme Court. I also make some adjustments, with the recent case of Texas v. Pennsylvania in mind.

The Justices should be unanimous in a refusal to hear a case of original jurisdiction, because there is no other avenue for such a case to be resolved, except the Fourth Box in Defense of Liberty, which the entire structure of government must be dedicated to avoiding. Even if they are unanimous, if more than one third of the states think it a worthy case, it must be resolved, not because it necessarily has merit, but because the integrity of the union is a risk if it is not resolved.